packages/scripts/src/generateIndex.ts

```diff
@@ -1,12 +1,17 @@
 import { stat, mkdir, writeFile } from 'node:fs/promises';
 import { join } from 'node:path';
 import { cwd } from 'node:process';
-import type { ApiPackage } from '@discordjs/api-extractor-model';
-import { ApiItem, ApiModel, ApiDeclaredItem, ApiItemContainerMixin, ApiItemKind } from '@discordjs/api-extractor-model';
+import type { ApiItem } from '@discordjs/api-extractor-model';
+import {
+	ApiPackage,
+	ApiModel,
+	ApiDeclaredItem,
+	ApiItemContainerMixin,
+	ApiItemKind,
+} from '@discordjs/api-extractor-model';
```

Comment:
Single import, if you really want to:

```suggestion
import {
	type ApiItem,
	ApiPackage,
	ApiModel,
	ApiDeclaredItem,
	ApiItemContainerMixin,
	ApiItemKind,
} from '@discordjs/api-extractor-model';
```